src/structures/paging/frame.rs

```diff
@@ -86,7 +86,6 @@ impl<S: PageSize> PhysFrame<S> {
 }
 
 impl<S: PageSize> fmt::Debug for PhysFrame<S> {
-    #[inline]
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         f.write_fmt(format_args!(
             "PhysFrame[{}]({:#x})",
```

Comment:
Could you clarify the reason for removing the `inline` here?

Reply:
For consistency. None of the other `Debug` implementations use `#[inline]`. (Except those that just forward to another `Debug` implementation.)

We could also make all the other `Debug` implementations `#[inline]` as well, but that doesn't seem to have any benefits.

Reply:
Thanks for clarifying, makes sense!